internal/ld/ld.go

```diff
@@ -91,10 +91,10 @@ func InitApiClient(options ApiOptions) ApiClient {
 	}
 }
 
-func (c ApiClient) GetFlagKeyList() ([]string, error) {
+func (c ApiClient) GetFlagKeyList(projKey string) ([]string, error) {
```

Comment:
Mentioned above. Now we explicitly pass in the project instead of relying on the `options.ProjKey`